mm/kfence/core.c

```diff
@@ -1,5 +1,7 @@
 // SPDX-License-Identifier: GPL-2.0
 
+#include <asm/pgalloc.h>
```

Comment:
Can this not be grouped with asm includes below? Does it pull in something needed by another include?

Reply:
Done.